src/services/Api.ts

```diff
@@ -14,21 +14,38 @@ interface Response<T> {
   data: T;
 }
 
+/**  Headers Definitions */
+
 const loginHeaders = { 'X-Auth-Type-Kiali-UI': '1' };
-const authHeader = (auth: string) => {
-  return { Authorization: auth };
+
+const authHeader = (auth: string) => ({ Authorization: auth });
+
+/**  Helpers to Requests */
+
+const getHeaders = (auth: string | undefined = undefined) => {
```

Comment:
arguments can be simplified: `(auth?: string)` , i think it works the same

Reply:
Yes, you are right

Reply:
Changed